vars/execSmokeTest.groovy

```diff
@@ -181,15 +179,26 @@ private def runPipeline(
         options['extraEnvVars']['DYNACONF_OCPROJECT'] = project
         options['namespace'] = project
 
+        def parameters = [
+            image: pipelineVars.iqeCoreImage,
+            namespace: project,
+            resourceLimitCpu: '1',
+            resourceLimitMemory: '2Gi',
+            cloud: options['cloud'],
+        ]
+
         runDeployStages(
             refSpec, project, ocDeployerBuilderPath, ocDeployerComponentPath,
-            ocDeployerServiceSets, buildScaleFactor, parallelBuild, options['cloud']
+            ocDeployerServiceSets, buildScaleFactor, deployScaleFactor,
+            parallelBuild, parameters
         )
 
         results = pipelineUtils.runParallel(iqeUtils.prepareStages(options, appConfigs))
 
-        stage("Collecting logs") {
-            openShiftUtils.collectLogs(project: project)
+        openShiftUtils.withNode(parameters) {
```

Comment:
I would do:

```
if (results['failed']) {
    <collect logs>
}
```

Since spinning up a new node can take ~1-2min (assuming everything is working OK on OpenShift) -- I think there's likely no need to spin up the node to collect logs if all the tests passed

Reply:
done